paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -372,7 +377,7 @@ class Paparazzi @JvmOverloads constructor(
   private fun scaleImage(image: BufferedImage): BufferedImage {
     val maxDimension = Math.max(image.width, image.height)
     val scale = THUMBNAIL_SIZE / maxDimension.toDouble()
-    return ImageUtils.scale(image, scale, scale)
+    return if (scale < 1f) ImageUtils.scale(image, scale, scale) else image
```

Comment:
I'd leave a comment on why we don't upscale the image.

Reply:
Fixed in [4b688ce](https://github.com/cashapp/paparazzi/pull/497/commits/4b688ce774b03c6bd538962618e867312b56dfbc)